js/src/kb-search/wp-seo-kb-search-init.js

```diff
@@ -0,0 +1,55 @@
+/* global wpseoAdminL10n */
+
+import React from "react";
+import ReactDom from "react-dom";
+import AlgoliaSearcher from "./wp-seo-kb-search.js";
+
+var intialiseAlgoliaSearch = () => {
```

Comment:
I think `initialize` in the functionname isn't describing what is done. @atimmer What do you think.